pkg/controller/embercsi/controller.go

```diff
@@ -95,7 +95,6 @@ func getControllerContainers(ecsi *embercsiv1alpha1.EmberCSI) []corev1.Container
 		args := []string{
 			"--v=5",
 			"--csi-address=/csi-data/csi.sock",
-			fmt.Sprintf("%s%s", "--provisioner=", GetPluginDomainName(ecsi.Name)),
```

Comment:
Won't this break deployments for older OCP/K8s where it is needed?

Reply:
This has been deprecated in 1.0.1 of the provisioner: https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-1.1.md

And 1.1 is the lowest sidecar for k8s >= 1.13 / OCP >= 4.1. Should be ok therefore?